_data/challenges.yml

```diff
@@ -123,16 +123,16 @@
       img: fa-gift
 
 - name: Desafio nº1
-  desc: <p>Este é o desafio que te vai dar mais trabalho, podes até ter de soar um pouco, mas vai valer a pena! Este desafio pode te até valer 30 pontos!</p><p>Método de classificação - tempo de cpu, desempate por rapidez de submissão</p></p>
+  desc: <p>Este é o desafio que te vai dar mais trabalho, podes até ter de suar um pouco, mas vai valer a pena! Este desafio pode te até valer 30 pontos!</p><p>Método de classificação - tempo de CPU, desempate por rapidez de submissão</p></p>
```

Comment:
If you are gonna fix, fix them all!
Can't you spot any other mistake on this description?